roles/openshift_manage_node/tasks/main.yml

```diff
@@ -59,12 +59,16 @@
   delegate_to: "{{ openshift_master_host }}"
 
 - name: Set node schedulability
-  command: >
-    {{ hostvars[openshift_master_host].openshift.common.client_binary }} adm manage-node {{ openshift.node.nodename }} --schedulable={{ 'true' if openshift.node.schedulable | bool else 'false' }}
-    --config={{ openshift_manage_node_kubeconfig }}
-    -n default
-  when: "'nodename' in openshift.node"
+  oadm_manage_node:
+    node: "{{ openshift.node.nodename | lower }}"
+    schedulable: "{{ 'true' if openshift.node.schedulable | bool else 'false' }}"
```

Comment:
I don't see a point of running the tasks unless schedulable is changing from false -> true. What is the point of changing false -> false?